README.md

```diff
@@ -30,7 +30,9 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
 
 2. Fork the project, using the top right corner button, that states `fork`
 
-3. Create a file named `your-username.json` in the directory [`profiles`](/profiles) with the following content.
+3. After forking the repository navigate to the folder with `cd <folder-name>` command
```

Comment:
I think your line is incomplete or repeating the same line number 3 context.
- Folder navigation is already mentioned in line number 3. as we can see -> directory [`profiles`](/profiles) 
- You have mentioned the `cd` command -> for using the `cd` command users have to clone the forked repository in their local machine.

You can document the steps well.
- As you have mentioned  `cd <folder-name>` and  haven't specified the specific folder name as in this case is [`profiles`](/profiles)

Reply:
Yes @kabir0x23 I'll update the steps which I've done